src/build/build.rs

```diff
@@ -59,59 +59,61 @@ fn main() {
 		format!("{}{}", env::var("OUT_DIR").unwrap(), "/built.rs"),
 	);
 
-	install_web_wallet();
+	let web_wallet_install = install_web_wallet();
+	match web_wallet_install {
+		Ok(true) => {}
+		_ => println!(
```

Comment:
Note that this message will [probably never be seen](https://github.com/rust-lang/cargo/issues/1106) and thus we should probably consider deleting messaging in any case where the build script will return 0 status code.